backend/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = ickly.test_settings
+addopts = --reuse-db
```

Comment:
There are probably some pytest config options you can use to skip data migrations and just seed it with your own test data or something, but this is fine for now. Again if you had VCR working this wouldn't be an issue. Let's work together to get that running soon.